packages/lumx-react/src/components/text/Text.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import { EllipsisType, GenericProps, truncate } from '@lumx/react/utils';
+
+/**
+ * Defines the props of the component.
+ */
+interface TextProps extends GenericProps {
+    /** The text to display */
+    text: string;
+    /** The max number of characteres to display */
+    maxChars?: number;
+    /** Works with maxChars. The ellipsis method to use */
+    ellipsisType?: EllipsisType;
```

Comment:
These props make sense only in the context of the truncate util so I would have create a unique prop:

```suggestion
    /** Defined how the text should be truncated. */
    truncate?: TruncateParams;
```

Because from my point of view, either you have a `Text` component with `text` and `truncate` props, or you have a `Truncate` component with `max`, `ellipsis` and `separator` props.